dev-identity/camunda-values.yaml

```diff
@@ -0,0 +1,100 @@
+# Chart values for the Camunda Platform 8 Helm chart.
+# This file deliberately contains only the values that differ from the defaults.
+# For changes and documentation, use your favorite diff tool to compare it with:
+# https://github.com/camunda/camunda-platform-helm/blob/main/charts/camunda-platform/values.yaml
+
+# This is a very small cluster useful for running locally and for development
+
+global:
+  image:
+    # allways work with the latest versions in development
+    tag: latest
+  identity:
+    auth:
+      # Disable the Identity authentication
+      # it will fall back to basic-auth: demo/demo as default user
+      enabled: true
+
+tasklist:
+  # Enabled if true, the tasklist deployment and its related resources are deployed via a helm release
+  enabled: true
+  # Env can be used to set extra environment variables on each Tasklist container
+  env: 
+    # Required to enabled tasklist API access while it is not enabled by default
+    - name: "SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI"
```

Comment:
This is not needed as the env variable SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI is being set by tasklist's spring application.yml based on the CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL which is generated by the tasklist's deployment.yaml. 
It should work simply by setting global.identity.auth.enabled: true and providing meaningful url for global.identity.auth.publicIssuerUrl

Reply:
Hi @McAlm - have you tried this? Or can you point me to where this is set? 

Please not that all logins work without the env vars. I am adding this specifically for tasklist API. This approach was the result of talking to several people. The conclusion seemed to be that Tasklist _should_ set the values by default, but doesn't. Of course it can be that we missed an alternative option.

Anyway, I tried your suggestion. When I set only:

```
global:
  image:
    # always work with the latest versions in development
    tag: latest
  identity:
    auth:
      enabled: true
      publicIssuerUrl: http://localhost:18080/auth/realms/camunda-platform
```
I get:
> 
>     Environment:
>       SPRING_PROFILES_ACTIVE:                           identity-auth
>       CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL:             http://localhost:18080/auth/realms/camunda-platform
>       CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL:     http://camunda-keycloak:80/auth/realms/camunda-platform
>       CAMUNDA_TASKLIST_IDENTITY_CLIENT_ID:              tasklist
>       CAMUNDA_TASKLIST_IDENTITY_CLIENT_SECRET:          <set to the key 'tasklist-secret' in secret 'camunda-tasklist-identity-secret'>  Optional: false
>       CAMUNDA_TASKLIST_IDENTITY_AUDIENCE:               tasklist-api
>       GRAPHQL_PLAYGROUND_ENABLED:                       true
>       GRAPHQL_PLAYGROUND_SETTINGS_REQUEST_CREDENTIALS:  include
>     Mounts:
>       /app/resources/application.yml from config (rw,path="application.yml")
>       /var/run/secrets/kubernetes.io/serviceaccount from kube-api-access-jhdc8 (ro)
> 

If I add the env vars explicitly I get:
>     Environment:
>       SPRING_PROFILES_ACTIVE:                               identity-auth
>       CAMUNDA_TASKLIST_IDENTITY_ISSUER_URL:                 http://localhost:18080/auth/realms/camunda-platform
>       CAMUNDA_TASKLIST_IDENTITY_ISSUER_BACKEND_URL:         http://camunda-keycloak:80/auth/realms/camunda-platform
>       CAMUNDA_TASKLIST_IDENTITY_CLIENT_ID:                  tasklist
>       CAMUNDA_TASKLIST_IDENTITY_CLIENT_SECRET:              <set to the key 'tasklist-secret' in secret 'camunda-tasklist-identity-secret'>  Optional: false
>       CAMUNDA_TASKLIST_IDENTITY_AUDIENCE:                   tasklist-api
>       GRAPHQL_PLAYGROUND_ENABLED:                           true
>       GRAPHQL_PLAYGROUND_SETTINGS_REQUEST_CREDENTIALS:      include
>       SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_ISSUERURI:  http://camunda-keycloak:80/auth/realms/camunda-platform
>       SPRING_SECURITY_OAUTH2_RESOURCESERVER_JWT_JWKSETURI:  http://camunda-keycloak:80/auth/realms/camunda-platform/protocol/openid-connect/certs

Reply:
Hi @rob2universe : The whole issue is being fixed here: https://github.com/camunda/camunda-platform-helm/pull/416 
Your approach was right, mine too. I changed the configmap for Tasklist to contain this section:
```
spring:
  config:
    activate:
      on-profile: "identity-auth"
  security:
    oauth2:
      resourceserver:
        jwt:
          issuer-uri: ${camunda.tasklist.identity.issuerUrl}
```

Reply:
Well, the Helm profiles should work with the unmodified official Helm chart. So let's use the extra env var until the issue is fixed.